studio/schemas/documents/compensations.ts

```diff
@@ -1,16 +1,31 @@
 import { defineField, defineType } from "sanity";
 import { titleSlug } from "../schemaTypes/slug";
 import seo from "../objects/seo";
+import offices from "../objects/offices";
 import { title } from "../fields/text";
 import { benefitId } from "./benefit";
 
+/*
+ maximum number of offices without preview string shortening
+ (should be at least 2)
+ given array: ["Trondheim", "Oslo", "Bergen", "Molde"]
+ 2 => Trondheim, and more
+ 3 => Trondheim, Oslo, and more
+ 4 => Trondheim, Oslo, Bergen, and Molde
+*/
+const OFFICES_PREVIEW_CUTOFF = 3;
```

Comment:
I'd keep it more simple:
```suggestion
export const compensationsId = "compensations";  // remember to keep sorting imports -> exports -> consts 

// Defines the maximum number of office locations to display in the preview without truncating
const OFFICES_PREVIEW_CUTOFF = 3;
```